examples/imagenet.py

```diff
@@ -0,0 +1,363 @@
+import os
+import warnings
+
+import hydra
+import torch
+import torch.nn as nn
+import torch.backends.cudnn as cudnn
+import torch.distributed as dist
+import torchvision.models as models
+import torchvision.datasets as datasets
+import torchvision.transforms as transforms
+from torch.optim import Adam
+
+best_acc1 = 0
+
+
+@hydra.main(config_name="imagenetconf")
```

Comment:
Did you forget to include the config?

Reply:
I wasn't sure how to use configen and I was mostly looking at `mnist_00.py`, and I couldn't find the corresponding configuration files for `mnist_00.py`. Should I just handcraft one and stick it in the example folder?

Reply:
Yes. configen is used to generate configs for libraries. for Examples and user code people should write their own configs.

Reply:
@briankosw

In `mnist_00.py`, I included the config in the top of the file.

https://github.com/pytorch/hydra-torch/blob/2656165d1b1f9a395e653d212742181b262e3986/examples/mnist_00.py#L20-L34

See the second code block of https://github.com/pytorch/hydra-torch/blob/master/examples/mnist_00.md#parting-with-argparse for the explanation!

Reply:
Generally, I think it is cleaner to make your own config python files and then import them. I simply included it in one flat file so that the tutorial reader can see it all in one place.

Reply:
This is the same thing in the draft `mnist_01.py` which I have yet to finalize. It's introducing some more hierarchical composition, so you'll see multiple 'non-configen' configs defined. Here's the whole 'hydra block':
https://github.com/pytorch/hydra-torch/blob/examples/mnist_01/examples/mnist_01.py#L10-L56

Reply:
Thanks! I'll do as you suggested.